db/migrate/20160405113943_drop_suplier_info.rb

```diff
@@ -0,0 +1,4 @@
+class DropSuplierInfo < ActiveRecord::Migration
+  def change
```

Comment:
sorry i forgot to mention the table name .it should be like this
drop_table  :suplier_infos inside change function

On Tue, Apr 5, 2016 at 6:24 PM, Kumaresan <notifications@github.com> wrote:

> In db/migrate/20160405113943_drop_suplier_info.rb
> <https://github.com/railsfactory-kumaresan/SampleApp/pull/2#discussion_r58532164>
> :
>
> > @@ -0,0 +1,4 @@
> > +class DropSuplierInfo < ActiveRecord::Migration
> > +  def change
>
> @amaninquirly <https://github.com/amaninquirly> Will this migration drop
> the suplier_infos table?. Explain how?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/railsfactory-kumaresan/SampleApp/pull/2/files/228eaa6a0da67c03d634ded2f479b6b2772067b2#r58532164>
>